library/Fission/Email/SendInBlue/TemplateId/Types.hs

```diff
@@ -0,0 +1,11 @@
+module Fission.Email.SendInBlue.TemplateId.Types (TemplateId(..)) where
+
+import Fission.Prelude
+
+
+newtype TemplateId = TemplateId { unTemplateId :: Int64 }
+  deriving          ( Generic )
```

Comment:
Generic?

Reply:
left over from when I didn't know what it was :grimacing: